site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -2656,6 +2658,7 @@ else if (state.getAttribute(STATE_CM_REQUESTED_SECTIONS) != null) {
 			if(daysafter > 0){
 				context.put("daysafter",daysafter);
 			}
+			DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss");
```

Comment:
Minor, but I _believe_ you could just use `withZone `here and avoid having to do `atZone `later. And might also be "nice to have" to have it final. I'm fine with your preference if you tested this to work. Thanks for the fix!

`final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss").withZone(localZoneId);`